deploy/roles/python-libs7/tasks/main.yml

```diff
@@ -38,3 +37,8 @@
   tags:
     - python-libs
 
+- name: Install Python packages for EPEL 36
+  command: pip3 install {{ ' '.join(python3_packages) }}
+  when: install_python
+  tags:
+    - python-libs
```

Comment:
It *might* fix the issue with slip/numpy to use the Ansible `pip` module instead of `command`, as that will install the packages sequentially, like so (see docs here: https://docs.ansible.com/ansible/latest/collections/ansible/builtin/pip_module.html):

```
- name: Install Python packages for EPEL 36
  pip:
    name: python3_packages
    executable: pip3
    umaks: 0022
  when: install_python
  tags:
    - python-libs
```